easybuild/easyconfigs/s/SPheno/SPheno-4.0.3-intel-2018a.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'ConfigureMake'
+
+name = 'SPheno'
+version = '4.0.3'
+
+homepage = 'http://spheno.hepforge.org/'
+description = """SPheno stands for S(upersymmetric) Pheno(menology). The code calculates 
+ the SUSY spectrum using low energy data and a user supplied high scale model as input"""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['http://www.hepforge.org/archive/spheno/']
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['43c0cb5934aa42dae0cbd7904a8895f963490817d3a1e9295c0d2884868b4da7']
+unpack_options = '--strip-components=1'
+
+skipsteps = ['configure', 'install']
+
+buildininstalldir = True
```

Comment:
just wondering, wouldn't the `MakeCp` easyblock be cleaner than `ConfigureMake` with `skipsteps = ['configure', 'install']` and `buildininstalldir = True`?

Reply:
I have also noticed that with the current approach the permissions in the installfolder are not ok

I will try the makecp easyblock

Reply:
@pescobar ping?